dask/dataframe/groupby.py

```diff
@@ -1171,6 +1171,26 @@ def max(self, split_every=None, split_out=1):
             token="max", func=M.max, split_every=split_every, split_out=split_out
         )
 
+    @derived_from(pd.DataFrame)
```

Comment:
Should this be `pd.core.groupby.GroupBy`?

Reply:
It probably should. I tried `pd.DataFrame` because it definitely has the `idxmin/max` methods, while `pd.core.groupby.GroupBy` doesn't declare them explicitly.

It doesn't fix the docs problem though...

Reply:
Changing this to `pd.core.groupby.GroupBy` makes the tests fail.